src/resources/extensions/quarto/video/video.lua

```diff
@@ -12,6 +12,17 @@ local interpolate = function(str, vars)
           end))
 end
 
+local function splitString (toSplit, delimiter)
+  if delimiter == nil then
+    delimiter = "%s"
+  end
```

Comment:
Lua tips useful for default argument in function and shorter syntax than full if then
```suggestion
  delimiter = delimiter or "%s"
```

Example: https://riptutorial.com/lua/example/4081/default-parameters

> The idiom `name = name or "Jack"` works because `or` in Lua short circuits. If the item on the left side of an `or` is anything other than `nil` or `false`, then the right side is never evaluated.

Reply:
This is better.  Thanks for suggesting.  Updated.